app/src/componentLibrary/breadcrumbs/breadcrumbs.jsx

```diff
@@ -23,47 +23,53 @@ import { Meatball } from './meatball';
 import styles from './breadcrumbs.scss';
 
 const cx = classNames.bind(styles);
+const BREADCRUMBS_WIDTH = 740;
+const MEATBALL_WIDTH = 36;
+const ARROW_WIDTH = 20;
 
-/* eslint-disable react/no-array-index-key */
-export const Breadcrumbs = ({ paths, url, dataAutomationId }) => {
-  const shownPaths = [...paths];
-  const { path: firstLevelPath, text: firstLevelPathText = firstLevelPath } = shownPaths[0];
+export const Breadcrumbs = ({ descriptors, titleTailNumChars, dataAutomationId }) => {
+  const shownDescriptors = [...descriptors];
```

Comment:
Why are you creating new value from prop?
Seems like it is read-only

Reply:
shownDescriptors.splice below is used which changes shownDescriptors.